snackager/k8s/base/deployment.yaml

```diff
@@ -21,9 +21,6 @@ spec:
         secret:
           secretName: git-account-credentials
           defaultMode: 0600
-      - name: redislabs-tls
-        secret:
-          secretName: redislabs-tls
```

Comment:
After deploying this, I think we'll need to manually delete the k8s Secret resource. It's not a big deal if we don't since it's not actually a sensitive value (public certificate) but good to clean up.